policyengine_us/variables/gov/states/ar/tax/income/credits/ar_additional_credit.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+class ar_additional_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arkansas additional credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AR
+
+    def formula(person, period, parameter):
+        taxable_income = add(tax_unit, period, ["ar_taxable_income_indiv"])
+        p = parameter(
+            period
+        ).gov.states.ar.tax.income.credits.additional
+        amount = p.amount
```

Comment:
```suggestion
        return p.amount.calc(taxable_income) 
        
```

Reply:
since `p.amount` is a scale parameter we will need to use the `.calc` function to attribute a value based on a determinant